packages/coil-extension/src/util/debounce.ts

```diff
@@ -0,0 +1,23 @@
+interface F {
+  (...args: any[]): void
+}
+
+export function debounce(fn: F, wait: number): F {
+  let args: any[] | null = null
+  let timer: NodeJS.Timer | null = null
+
+  function flush(): void {
+    if (args !== null) fn(...args)
+    args = timer = null
+  }
+
+  return function (...rest): void {
```

Comment:
How is (...rest) getting past `"noImplicitAny": true` in the tsconfig ??

Reply:
I don't know. Maybe since it is `any[]`, not `any`?